src/lang/en-US.json

```diff
@@ -676,6 +678,21 @@
   "story_gold": "gold",
   "story_chat_asked": "asked",
   "story_chat_said": "said",
+  "subscriptions_h1": "Subscribe",
+  "subscriptions_h2":"Support open source development and the Dota 2 data community",
+  "subscriptions_body1":"In 2015 when we first started OpenDota, then YASP, we had one mission in mind: make advance statistics available to the entire Dota community, free of charge and open sourced. Our goal has never been to make money; we have always minimized ads on the site. Howevever, as our servers costs vontinue to increase (storage is expensive!) and sponsorships churn, we must move to a more sustainable funding model to keep this mission alive.",
+  "subscriptions_body2":" Become a subscriber for just $5 a month and all your matches will be automatically parsed. For users who are unable to pay, you can continue to manually request matches for parsing:",
+  "subscriptions_request": "Request a parse",
```

Comment:
Don't know if we need to specifically link the request page on the subscribe page